src/main/java/com/plaid/client/model/paymentinitiation/Options.java

```diff
@@ -0,0 +1,37 @@
+package com.plaid.client.model.paymentinitiation;
+
+public class Options {
```

Comment:
I think it might make more sense to either fold this options class into the PaymentInitiationCreate class or name it something different since different payment initiation requests could have different options

Reply:
That makes sense since we may introduce options for recipient or other entities.  How do you feel about renaming it to `PaymentOptions`?